lib/newProfile.ml

```diff
@@ -16,11 +16,27 @@ module MiniJson = struct
     | `List of t list
   ]
 
+  (* https://www.ietf.org/rfc/rfc4627.txt *)
```

Comment:
You may want to add a link to the yojson version, and maybe reorder the match to follow it?

This way we can quickly check if we have deviated.